logging_json/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "JSON Logging",
-    "version": "18.0.1.0.0",
+    "version": "19.0.1.0.0",
```

Comment:
AFAIK this is outdated. See
https://github.com/camptocamp/odoo-cloud-platform/pull/502
https://github.com/camptocamp/odoo-cloud-platform/pull/508

But I don't know if it is installed or not on instances